docs/content/en/blog/news/announcing-pipecd/index.md

```diff
@@ -102,4 +102,4 @@ PipeCD team is hiring engineers/interns to work on PipeCD. Please contact us if
 ### Thanks
 
 Finally, we would like to thank the existing open-source CD projects like Spinnaker, FluxCD, ArgoCD... PipeCD has been built on many great ideas from those great projects.
-PipeCD team would also like to thank CyberAgent's engineers and collaborators from other companies, who have sent us so many valuable feedback throughout the development process to this day.
+PipeCD team would also like to thank CyberAgent's engineers and collaborators from other companies, who have sent us so many valuable feedbacks throughout the development process to this day.
```

Comment:
"feedback" is uncountable.

Reply:
thanks, addressed on https://github.com/pipe-cd/pipe/pull/1111/commits/8ff128265b94202cfac3c8b48ca724da50739740.